configure.ac

```diff
@@ -40,9 +40,21 @@ AS_IF([test "x$with_cddlib" != "xyes"],[
   CDD_CPPFLAGS="-I$with_cddlib/include/cdd -I$with_cddlib/include/cddlib -I$with_cddlib/include"
   CDD_LDFLAGS="-L$with_cddlib/lib -lcddgmp"
 ],[
-  CDD_CPPFLAGS="-I/usr/include/cdd"
+  CDD_CPPFLAGS="-I/usr/include/cdd -I/usr/include/cddlib"
   CDD_LDFLAGS="-lcddgmp"
 ])
+
+dnl ##
+dnl ## Detect Cygwin, as then we need to link against gmp
+dnl ##
+AC_CANONICAL_HOST
+
+case $host_os in
+  *cygwin* ) CDD_LDFLAGS="$CDD_LDFLAGS -lgmp";;
```

Comment:
Actually this is also needed on macOS. So I made PR #31 as an alternative to this one.